operations/research-and-development/product/product-planning/how-to-use-productboard.md

```diff
@@ -6,18 +6,24 @@ description: 'https://mattermost.productboard.com/'
 
 Productboard is used by Product Managers and Customer/Sales Teams for:
 
-- Feedback collection
-- Feature validation
-- Requirements gathering
-- Feature prioritization/roadmap planning
+* Feedback collection
+* Feature validation
+* Requirements gathering
+* Feature prioritization/roadmap planning
 
 When a feature moves to **Planned** the Product team pushes it to Confluence and Jira (or wherever each team decides) to define the scope/spec and track work.
 
 Specifically, productboard is used to decide what to build (and when), while other systems like Confluence or Jira are used to communicate and design on what exactly is being built at individual feature level.
 
-## **1 - Day in the life of a Product Manager**
+**Roles**
 
-### **1a- Reviewing new feedback from users and customers**
+**Makers:** Product Managers and Product Management leadership
```

Comment:
The roles are not rendering in a list format.

Reply:
Should be fixed now.